pkg/node_exporter/Dockerfile

```diff
@@ -1,7 +1,28 @@
-FROM prom/node-exporter:v0.15.1@sha256:88c602650e861bb8045ac83421edecbd056fc58708d47554a0d1a10f19ed7556
+FROM linuxkit/alpine:07f7d136e427dc68154cd5edbb2b9576f9ac5213 as build
+
+RUN apk add --no-cache go git g++ make
```

Comment:
is `g++` needed? Other pure go packages tend to just install `go musl-dev`, but of course you also need `make` here because it's used below.